jberet-core/src/main/java/org/jberet/repository/InfinispanRepository.java

```diff
@@ -183,6 +183,12 @@ public List<JobExecution> getJobExecutions(final JobInstance jobInstance) {
         return result;
     }
 
+    // todo
```

Comment:
Maybe I can move the `InfinispanRepostiory` and the `MongoRepository` out of `jberet-core` to reduce the depdencies. That's another task though.